src/App.jsx

```diff
@@ -14,70 +16,86 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const DEFAULT_GOOD = goods[8];
+  const [titleGood, setTitleGood] = useState(DEFAULT_GOOD);
+  const [noGoodMessage, setNoGoodMessage] = useState(false);
```

Comment:
There is no need to create a separate state from it. Just check if the selectedGood is not empty